floor/floor/controller/playlist.py

```diff
@@ -16,7 +16,7 @@ def __init__(self, filename=None, processor=None):
 
         # If a processor was passed in, use it, otherwise read the config file
         if processor:
-            self.append(processor)
+            self.append(processor.__name__)
```

Comment:
This breaks the `--processor` flag since it sends in a string.  Should that flag construct the processor object or should this code send just the name in?

Reply:
Oh! Yeah, this has been broken for a while apparently - `.append()` wants a name. So we could simply change the constructor to take a name (and rename the arg to `processor_name` for good measure).

If that works for you, I'll make the change!

Reply:
Sure that sounds good